Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -97,6 +107,7 @@ export default function widgetBehavior(publicAPI, model) {
     ) {
       model.previousPosition = calldata.position;
       isScrolling = true;
+      publicAPI.resetUpdateMethod();
       publicAPI.startInteraction();
```

Comment:
it looks like you can factorize some code:
```
startScrolling(newPosition) {
if (newPosition) {
  model.previousPosition = newPosition;
}
isScrolling = True;
publicAPI.resetUpdateMethod();
publicAPI.startInteraction();
}
```

Reply:
Done.
Before you'll ask, I can't set the `isScrolling` to true in `handleStartMouseWheel` because, if you set it to true, then you scroll and move the mouse at the same time, it will try to access to `handleMouseMove` and will potentially failed, especially if you scroll right after loading data, because model.previousPosition doesn't exist.

Reply:
Does it look better ?